src/middleware/externalRequests.js

```diff
@@ -44,6 +44,16 @@ const performRequests = (requests, ctx) => {
     // No body is sent out for now
     const body = null
 
+    if (ctx.request.header['x-openhim-transactionid']) {
```

Comment:
@MattyJ007 I have done the initial review and everything looks great. Could you move the OpenHIM Transaction ID header string value to a `const` to help avoid typos when using it.